public/logos/1860/SC.svg

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
can you remove the white spaces in these svgs?

Reply:
Sure. I created these with inkscape, but I'll look for a tool to remove the whitespace.